src/typeUtils.ts

```diff
@@ -185,17 +186,17 @@ export interface RenderProps extends RouteMatch {
  * @see https://github.com/4Catalyzer/found/blob/master/README.md#render
  */
 export interface RouteRenderArgs {
-  match: Match;
+  match: Match | RouteMatch;
```

Comment:
i wonder if we should keep this as Match and maybe cast the cases where it isn't?